spec/datadog/tracing/contrib/support/tracer_helpers.rb

```diff
@@ -68,8 +68,8 @@ def clear_traces!
       config.before do
         # DEV `*_any_instance_of` has concurrency issues when running with parallelism (e.g. JRuby).
         # DEV Single object `allow` and `expect` work as intended with parallelism.
-        allow(Datadog::Tracing::Tracer).to receive(:new).and_wrap_original do |method, *args, &block|
-          instance = method.call(*args, &block)
+        allow(Datadog::Tracing::Tracer).to receive(:new).and_wrap_original do |method, **args, &block|
+          instance = method.call(**args, &block)
```

Comment:
Minor: make it explicit that this is keyword arguments now.

```suggestion
        allow(Datadog::Tracing::Tracer).to receive(:new).and_wrap_original do |method, **kwargs, &block|
          instance = method.call(**kwargs, &block)
```

Also, wondering what happened to `*args`: is is not needed anymore?

My spider-sense is tingling here WRT the argument changes that happened between 2.6, 2.7, 3.0+3.1, and 3.2.

Reply:
Note that this is a specific simpler fix for the `Tracer` class only -- if this was something on some outside gem we'd need to sprinkle our friend `ruby2_keywords` instead ;)

What happened here is that on dd-trace-rb 0.x, this used to be the classic `options = {}`:

https://github.com/DataDog/dd-trace-rb/blob/v0.54.2/lib/ddtrace/tracer.rb#L76

...but on 1.0 it was moved to use real keyword arguments:

https://github.com/DataDog/dd-trace-rb/blob/v1.0.0/lib/datadog/tracing/tracer.rb#L50

Up to 3.2, the forwarding we were doing here still worked, but it stopped working in 3.2. And for all the others, this change works because the class gets keyword arguments.